File: C/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int c25 = 0;
    int c10 = 0;
    int c5 = 0;
    int c1 = 0;
    float n;

    do
    {
        n = get_float("Change: ");
    }
    while (n < 0);

    int cents = round(n * 100);

    c25 = cents / 25;
    c10 = (cents - (c25 * 25)) / 10;
    c5 = (cents - (c25 * 25) - (c10 * 10)) / 5;
    c1 = cents - (c25 * 25) - (c10 * 10) - (c5 * 5);

    printf("%i\n", c25 + c10 + c5 + c1);
}

File: Python/cash/cash.py
c25 = 0
c10 = 0
c5 = 0
c1 = 0

n = 0
while (n <= 0):
    n = float(input("Change: "))
cents = round(n * 100)

c25 = cents // 25
cents %= 25

c10 = cents // 10
cents %= 10

c5 = cents // 5
cents %= 5

c1 = cents // 1

total = c25 + c10 + c5 + c1

print(f"Total: {total}")
